samples/androidNativeActivity/src/main/kotlin/engine.kt

```diff
@@ -160,7 +164,7 @@ class Engine(val arena: NativePlacement, val state: NativeActivityState) {
     }
 
     private fun getEventPoint(event: CPointer<AInputEvent>?, i: Int) =
-            Vector2(AMotionEvent_getRawX(event, i.signExtend<size_t>()), AMotionEvent_getRawY(event, i.signExtend<size_t>()))
+            Vector2(AMotionEvent_getRawX(event, i.signExtend<platform.posix.size_t>()), AMotionEvent_getRawY(event, i.signExtend<platform.posix.size_t>()))
```

Comment:
Likely <platform.posix.size_t> part is not needed.

Reply:
done